### PR TITLE
Add DAG for monitoring tasks

### PR DESCRIPTION
**Pull request by scholtzan**
For now this DAG only schedules one task for determining partition sizes but I guess there will be more monitoring related tasks that can be added here in the future.

### CODE REVIEW

File: dags/monitoring.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+from airflow.operators.sensors import ExternalTaskSensor
+from datetime import timedelta, datetime
+from operators.gcp_container_operator import GKEPodOperator
+from utils.gcp import gke_command
+
+default_args = {
+    "owner": "ascholtz@mozilla.com",
+    "email": ["ascholtz@mozilla.com"],
+    "depends_on_past": False,
+    "start_date": datetime(2020, 7, 1),
```
Comment (scholtzan) [inline dags/monitoring.py:11]: I'm not sure if we also want to backfill some of this data, then we can set it to something earlier. Or later

Comment (jklukas) [reply dags/monitoring.py:11]: I do think it could be useful to backfill through the full history we have for main_v4, back to 2018-10-30.

Comment (scholtzan) [reply dags/monitoring.py]: 👍  Updated the timestamp

**Comment by scholtzan** [comment]
I guess this is ready to go?

**Pull request merged**